### PR TITLE
Update docs for MC-1285

### PR DESCRIPTION
**Pull request by aberatcan**
- updated docs for newly added env property

### CODE REVIEW

File: docs/modules/deploy-manage/pages/system-properties.adoc [hunk 1]
```diff
@@ -76,6 +76,14 @@ hz-mc start -Dhazelcast.mc.contextPath=hazelcast-mc \
 
 In this example, the URL for Management Center would be `\http:localhost:8080/hazelcast-mc`.
 
+|[[hazelcast-mc-cluster-operation-timeout]]hazelcast.mc.cluster.operation.timeout.millis
+|Operation timeout in milliseconds for scripts in MC. Default: 60000.
```
Comment (JakeSCahill) [inline docs/modules/deploy-manage/pages/system-properties.adoc:80]: Could we be a bit more descriptive? I'm not sure what causes the timeout. Maybe a link to the scripting page would also help.

Is this the number of milliseconds that a script can be unable to connect to a cluster?

**Review by JakeSCahill — CHANGES_REQUESTED** [review]

File: docs/modules/deploy-manage/pages/system-properties.adoc (hunk 1 above)
Comment (promanenko) [reply docs/modules/deploy-manage/pages/system-properties.adoc:80]: @aberatcan this timeout applies to all MC operations on the members, not only to scripts execution.
@JakeSCahill a bit of context https://hazelcast.slack.com/archives/C9S7BV74Z/p1642071001005500?thread_ts=1642070781.005000&cid=C9S7BV74Z

Comment (JakeSCahill) [reply docs/modules/deploy-manage/pages/system-properties.adoc:80]: Thanks @promanenko 😄 

In that case: 'Timeout in milliseconds for Management Center operations that wait for a response from members.'?

Also, this is a troubleshooting topic. Could we add an entry to this page? https://github.com/hazelcast/management-center-docs/blob/main/docs/modules/troubleshoot/pages/management-center.adoc

This is where we can discuss what to do if you experience a timeout when running scripts or SQL queries.

File: docs/modules/troubleshoot/pages/management-center.adoc [hunk 2]
```diff
@@ -42,4 +42,9 @@ copy the log entries to the clipboard using **Copy**
...
 
 NOTE: You may only need to share the information in these logs
-if requested by the Hazelcast's support team.
\ No newline at end of file
+if requested by the Hazelcast's support team.
+
+== Experience Timeout
+
+If you experience a timeout when running scripts or SQL queries, You can increase operation timeout value via setting
+new value to xref:deploy-manage:system-properties.adoc#hazelcast-mc-cluster-operation-timeout['hazelcast.mc.cluster.operation.timeout.millis'] property.
```
Comment (JakeSCahill) [inline docs/modules/troubleshoot/pages/management-center.adoc:50]:
```suggestion
in the xref:deploy-manage:system-properties.adoc#hazelcast-mc-cluster-operation-timeout[`hazelcast.mc.cluster.operation.timeout.millis`] property.
```

File: docs/modules/troubleshoot/pages/management-center.adoc [hunk 3]
```diff
@@ -42,4 +42,9 @@ copy the log entries to the clipboard using **Copy**
 and resume the flow using **Resume**.
 
 NOTE: You may only need to share the information in these logs
-if requested by the Hazelcast's support team.
\ No newline at end of file
+if requested by the Hazelcast's support team.
+
+== Experience Timeout
+
+If you experience a timeout when running scripts or SQL queries, You can increase operation timeout value via setting
```
Comment (JakeSCahill) [inline docs/modules/troubleshoot/pages/management-center.adoc:49]:
```suggestion
If you experience a timeout when running scripts or SQL queries, you can increase the timeout value 
```

**Review by JakeSCahill — CHANGES_REQUESTED** [review]
Just some minor sentence structure changes.

**Review by JakeSCahill — APPROVED** [review]

**Pull request merged**